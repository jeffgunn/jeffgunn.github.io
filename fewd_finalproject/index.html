<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Wrecking Bar</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="css/normalize.css">
    <link rel="stylesheet" type="text/css" href="css/style.css">
    <script src="https://use.fontawesome.com/d8a95c750f.js"></script>
    <link href="https://fonts.googleapis.com/css?family=Droid+Serif:400,700|Open+Sans:400,600,700" rel="stylesheet">
  </head>

  <body>
    <div class="container">
      <header>
    		<div id="logo">
    			<a href="#"><img src="images/logo-black.png" alt="Wrecking Bar Brewpub" width="180px"> </a>
        </div>
        <nav>
          <span class="menu-trigger">MENU</span>
      		<div class="nav-menu">
      			<ul>
      				<li><a href="#">Menu</a></li>
      				<li><a href="#">Brews</a></li>
      				<li><a href="#">Librations</a></li>
      				<li><a href="#">Venue</a></li>
      				<li><a href="#">Calendar</a></li>
              <li><a href="#">About</a></li>
      			</ul>
          </div>
        </nav>
      </header>
      <!-- /nav-menu -->
      <section id="slideshow">
        <div>
          <h2>Home Page Slider</h2>
          <p>Lorem ipsum dolor sit amet</p>
        </div>
      </section>
  		<!-- main-content -->
  		<div class="main-content">
  			<h2>Main Content</h2>
  			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
  			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
  			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
  			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
  			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
  		</div>
    </div>
    <script type="text/javascript" src="js/jquery-1.11.0.min.js"></script>
  	<script type="text/javascript" src="js/script.js"></script>
  </body>
</html>
