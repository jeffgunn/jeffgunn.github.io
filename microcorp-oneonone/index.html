<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Microcorp: The 16th Annual One on One</title>

    <!-- Bootstrap core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <link href="css/ie10-viewport-bug-workaround.css" rel="stylesheet">
    <!-- Custom styles for this template -->
    <link href="css/starter-template.css" rel="stylesheet">
    <link href="css/micro.css" rel="stylesheet">
    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="css/ie8-responsive-file-warning.js"></script><![endif]-->
    <script src="js/ie-emulation-modes-warning.js"></script>
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
 <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
 <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

  <body>

<nav class="navbar navbar-inverse navbar-fixed-top">
    <div class="container">
		 <div class="navbar-header">
		   <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
			 <span class="sr-only">Toggle navigation</span>
			 <span class="icon-bar"></span>
			 <span class="icon-bar"></span>
			 <span class="icon-bar"></span>
		   </button>
		   <!--<a class="navbar-brand" href="#">Project name</a>-->
		 </div>
		 <div id="navbar" class="collapse navbar-collapse">
		   <ul class="nav navbar-nav">
			 <li><a href="#schedule">Schedule</a></li>
			 <li><a href="#venue">Venue</a></li>
			 <li><a href="#transport">Transportation</a></li>
			 <li><a href="#directions">Directions</a></li>
			 <li><a href="#register">Register</a></li>
		   </ul>
		 </div><!--/.nav-collapse -->
    </div>
  </nav>

	<div class="container no-padding" id="desktop-banner">
	  <div class="row">
	    <div class="col-xs-12">
	 		<img src="images/masthead.jpg" alt="Microcorp: The 16th Annual One on One - The Ultimate Partnership Play | Omni CNN Center - June 12-14, 2019" class="img-responsive" />
	    </div>
	  </div>
	</div>

	<div class="container no-padding" id="mobile-banner">
	  <div class="row">
	    <div class="col-xs-12">
	 		<img src="images/masthead-mobile.jpg" alt="Microcorp: The 16th Annual One on One - The Ultimate Partnership Play | Omni CNN Center - June 12-14, 2019" class="img-responsive" />
	    </div>
	  </div>
	</div>

  <a id="schedule" name="schedule">
	<div class="container">
    <div class="row">
        <div class="col-xs-12 section-title">SCHEDULE</div>
    </div>
	</div>
	</a>

  <!--Content-->
      <div class="container section-content">



        <div class="row">
          <div class="col-md-6 sched-block">
            <h2 style="text-align: center">Wednesday, June 12, 2019</h2>
            <div class="sched-content">
                <div class="sched-time">Noon - 6:00 pm</div>
                <div class="sched-act">Registration Open</div>
                <div class="sched-location">Omni Atlanta Hotel  - South Tower Lobby</div>
            </div>
            <div class="sched-content">
                <div class="sched-time"><strong>1:00 PM - 2:00 PM</strong></div>
                <div class="sched-act">TBD Sessions</div>
                <div class="sched-location">Cypress</div>
            </div>
			<div class="sched-content">
                <div class="sched-time"><strong>2:00 PM - 3:00 PM</strong></div>
                <div class="sched-act">TBD Sessions</div>
                <div class="sched-location">Cypress</div>
            </div>
            <div class="sched-content">
                <div class="sched-time">3:00 pm - 5:30 pm</div>
                <div class="sched-act">General Session</div>
                <div class="sched-location">International Ballroom E/F</div>
            </div>
            <div class="sched-content">
                <div class="sched-time"><strong>5:30 PM - 6:30 PM</strong></div>
                <div class="sched-act">New Partner Meet &amp; Greet</div>
				        <div class="sched-location">Juniper</div>
            </div>
            <div class="sched-content-pic">
              <img src="images/photo-hof1.jpg" alt="College Football Hall of Fame" class="img-responsive" />
            </div>
            <div class="sched-content">
                <div class="sched-time"><strong>6:30 PM - 10:30 PM</strong></div>
                <div class="sched-act">Opening Event</div>
                <div class="sched-location">College Football Hall of Fame</div>
<!--
                <div class="sched-smallpics">
                  <img src="images/photo-hof2.jpg" alt="College Football Hall of Fame" class="img-hof-thumb-left" />
                  <img src="images/photo-hof3.jpg" alt="College Football Hall of Fame" class="img-hof-thumb-right" />

                </div>
-->
            </div>
            <div class="sched-content sched-content-end">
                <div class="sched-time"><strong>10:30 PM</strong></div>
                <div class="sched-act">Post Event Cocktails</div>
                <div class="sched-location">Omni Atlanta Hotel Lobby – South Tower Lobby</div>
            </div>
          </div>


          <div class="col-md-6 sched-block">
            <h2 style="text-align: center">Thursday, June 13, 2019</h2>
            <div class="sched-content">
                <div class="sched-time">7:30 am - 8:30 am</div>
                <div class="sched-act">Breakfast</div>
                <div class="sched-location">Internal Ballroom E/F</div>
            </div>
            <div class="sched-content">
                <div class="sched-time">8:30 am - 9:45 am</div>
                <div class="sched-act">General Session</div>
                <div class="sched-location">International Ballroom E/F</div>
            </div>
            <div class="sched-content-break">
                <div class="sched-time">9:45 am - 10:00 am</div>
                <div class="sched-act">Break</div>
            </div>
            <div class="sched-content">
                <div class="sched-time"><strong>10:00 AM - 12:00 PM</strong></div>
                <div class="sched-act">One on One Meetings</div>
                <div class="sched-location">International Ballroom C/D</div>
            </div>
            <div class="sched-content">
                <div class="sched-time"><strong>12:00 PM - 1:00 PM</strong></div>
                <div class="sched-act">Lunch &amp; Awards</div>
                <div class="sched-location">International Ballroom E/F</div>
            </div>
            <div class="sched-content">
                <div class="sched-time">1:00 pm - 3:00 pm</div>
                <div class="sched-act">General Session</div>
                <div class="sched-location">International Ballroom E/F</div>
            </div>
            <div class="sched-content-break">
                <div class="sched-time">3:00 pm - 3:15 pm</div>
                <div class="sched-act">Break</div>
            </div>
            <div class="sched-content-break">
                <div class="sched-time">5:00 pm - 5:15 pm</div>
                <div class="sched-act">Break</div>
            </div>
            <div class="sched-content">
                <div class="sched-time">5:15 pm - 7:15 pm</div>
                <div class="sched-act">Expo</div>
                <div class="sched-location">Atrium Terrace A</div>
            </div>
            <div class="sched-content-break">
                <div class="sched-time">7:15 pm - 7:30 pm</div>
                <div class="sched-act">Break</div>
            </div>
            <div class="sched-content sched-content-end">
                <div class="sched-time">7:30 pm - 10:30 pm</div>
                <div class="sched-act">Recognition Banquet</div>
                <div class="sched-location">International Ballroom E/F</div>
            </div>
            
          </div>
        </div>
      </div>
	  
	  <div class="container section-content">
        <div class="row">
            <div class="col-xs-12">
				<h2 style="text-align: center">Friday, June 14, 2019</h2>
				<div class="sched-content sched-content-end">
					<div class="sched-time">9:00 am - 10:30 am</div>
					<div class="sched-act">Farewell Breakfast Session</div>
					<div class="sched-location">International Ballroom E/F</div>
				</div>
			</div>
        </div>
    </div>


    <a id="venue"><div class="container">
        <div class="row">
            <div class="col-xs-12 section-title">Venue</div>
        </div>
    	</div>
	</a>

    <div class="container section-content section-grade">
      <div class="row">
   		<div class="col-md-6 col-md-push-6"><img src="images/photo-omni.jpg" alt="Omni Hotel" class="img-responsive img-thumbnail" /></div>
  		 <div class="col-md-6 col-md-pull-6">
		<h2>Omni Atlanta Hotel at CNN Center</h2>
       <p>100 CNN Center <br>
Atlanta, GA 30303<br>
Phone:  404-659-0000</p>
       <p><strong>GPS: Enter the address:  <br>
         190 Marietta Street, Atlanta, GA  30303  </strong></p>
       <p><strong>Arrival:</strong><br>
         Noon on June 12th</p>
       <p><strong>Departure:</strong><br>
         10:00 AM on June 14th</p>
       <p><strong>Hotel Reservations:</strong><br>
         To take advantage of the group event rate, hotel reservations must be made by February 28, 2019 through the event registration site. Confirmations will be sent to registered attendees in early June.</p>
       <p><strong>Event Support:</strong><br>
         Jenn Cannon: 770-310-4558 or <a href="mailto:MicroCorpMeetingHQ@ElevateIME.com">MicroCorpMeetingHQ@ElevateIME.com</a>  	</p>
	    </div>
  	</div>
	</div>


	<a id="transport"><div class="container">
        <div class="row">
            <div class="col-xs-12 section-title">Transportation</div>
        </div>
    	</div>
	</a>

    <div class="container section-content section-grade">
      <div class="row">
      </div>
      <div class="row">
	    <div class="col-md-12">
		   <h2>Transportation from Hartsfield-Jackson Atlanta International Airport (ATL):</h2>
               <p><strong>Taxi (Recommended)</strong>  = Estimated fare = $32 USD</p>
               <p><strong>Uber/Lyft (Recommended)</strong> = Estimated fare = $15 USD (non-surge rate)</p>

		   <h2>MARTA</h2>

		  <p><strong>$3.50</strong></p>
		  <ul>
			  <li>From your terminal, take the Plane Train to the Baggage Claim stop</li>
			  <li>Exit at Baggage Claim, then take elevator to the top floor </li>
			  <li>From the top floor, follow signs to &ldquo;MARTA Train Station&rdquo; • You can go left or right at this junction- both ways will lead you to MARTA Station </li>
			  <li>Stop by the Breeze Card Kiosk and purchase your Marta Breeze Pass ($3.50). </li>
			  <li>Scan your Breeze Pass, then take elevator up to the train	    </li>
		  </ul>
		  <h2>Parking</h2>
		  <p>The Omni Atlanta is located in the heart of Downtown Atlanta. Multiple parking options are available within the neighborhood. <a href="pdf/parking.pdf">Download the parking map</a>.</p>
		  <p><strong>Valet Parking at the Omni Atlanta</strong><br>
	      The Omni Atlanta offers Event Valet Parking at $20/vehicle and is subject to availability.</p>
		  <p><strong>Nearby Self Parking Options Include</strong><br>
          <a href="https://www.google.com/maps/place/Marietta+Deck/@33.7606501,-84.3964202,19z/am=t/data=!3m1!4b1!4m5!3m4!1s0x88f5047fdbe90317:0xe043ed4d36c6ab89!8m2!3d33.7606501!4d-84.395873" target="_blank">Marietta Parking Deck</a> ($10) 258 Marietta Street NW Atlanta, GA 30303<br>
          <a href="https://www.google.com/maps/place/113+Centennial+Olympic+Park+Dr+NW,+Atlanta,+GA+30303/@33.7571182,-84.3967202,17z/data=!3m1!4b1!4m5!3m4!1s0x88f503804158bbe1:0x26c07766965d5d69!8m2!3d33.7571182!4d-84.3945315?q=113+Centennial+Olympic+Park+Dr+NW+Atlanta,+GA+30303&rlz=1C1KMZB_enUS724US724&um=1&ie=UTF-8&sa=X&ved=0ahUKEwiwnMSbs-jTAhWD6IMKHbQMBf0Q_AUICigB" target="_blank">CNN Parking Deck</a> ($10) 113 Centennial Olympic Park Dr NW Atlanta, GA 30303</p>
		  <p><em><strong>Please Note:</strong> parking options not owned or managed by Omni Hotels. Pricing and availability subject to change without notice </em><br>
		  </p>
		  <p>Additional Options<br>
	      <a href="http://atlanta.bestparking.com/destinations/cnn-center-parking" target="_blank">Best Parking</a>, <a href="https://www.parkingpanda.com/omni-hotel-atlanta-parking?ref=omni" target="_blank">Parking Panda</a>, <a href="https://www.uber.com/events/" target="_blank">Uber</a></p>
	    </div>
  	</div>
	</div>

  <a id="directions"><div class="container">
        <div class="row">
            <div class="col-xs-12 section-title">Directions</div>
        </div>
    	</div>
	</a>

    <div class="container section-content section-grade">
      <div class="row">
      </div>
      <div class="row">
  		 <div class="col-md-12">
			 <h2>Driving Directions</h2>
               <p><strong>GPS ADDRESS</strong><br>
           When following GPS, please use <a href="https://goo.gl/maps/12jrtyKFRTs" target="_blank">190 Marietta St NW Atlanta, GA 30303</a></p>
           <p><strong>Directions from I-75 North &amp; I-85 NORTH (Traveling North from the Hartsfield-Jackson Atlanta International Airport) – 12 miles </strong> <br>
           Take I-85 N (merges with I-75 N) to the International Boulevard Exit (exit 248C). At the second light at the top of the exit ramp, turn left onto Andrew Young International Boulevard. Then turn left onto Centennial Olympic Park Drive. Take the first right on Marietta Street. The Motor Lobby entrance is 1 block on the left side of Marietta Street.</p>
               <p><strong>Directions from I-75 South &amp; I-85 (Driving South)</strong> <br>
           Follow I-75/85 S into downtown Atlanta. Take Exit 249-C (Williams Street) and continue straight at the exit. Go to the fifth traffic light and turn right onto Andrew Young International Boulevard. Turn left onto Centennial Olympic Park Drive and go 3 blocks to Marietta Street and turn right. The Hotel Motor Lobby will be on your left.</p>
           <p><strong>Directions from the East – Via I-20</strong> <br>
           Take I-20 West to Exit 56B (Windsor Street/Spring Street). Turn right onto Spring Street. Spring Street will turn into Ted Turner Drive, continue on Ted Turner Drive until you reach Marietta Street. Turn left onto Marietta Street and go 2 blocks to Andrew Young International Boulevard. Turn left into the Hotel Motor Lobby. </p>
           <p><strong>Directions from the West – Via I-20<br>
           </strong>Take I-20 East to Exit 56B (Windsor Street/Spring Street). Turn left onto Spring Street. Spring Street will turn into Ted Turner Drive, continue on Ted Turner Drive until you reach Marietta Street. Turn left onto Marietta Street and go 2 blocks to Andrew Young International Boulevard. Turn left into the Hotel Motor Lobby.
		   </p>
           <p><strong>Parking<br>
</strong>Valet Covered Parking is available for $45.00 per night and includes in/out privileges.</p>

		  </div>
    	</div>

			 <div class="row">
   		<div class="col-md-6 col-md-push-6"><img src="images/photo-martamap.jpg" alt="MARTA Walking Map" class="img-responsive img-thumbnail" /></div>
  		 <div class="col-md-6 col-md-pull-6">
           <h2>MARTA Directions		   </h2>
			 <ol start="1" type="1">
		     <li>Take a blue or green line train to the GWCC/CNN station. (If you are      traveling&nbsp;from the airport or any station on the red or gold rail      line, travel to the Five Points station and transfer to the westbound      train platform up one level. Board the first westbound train that arrives      and ride one stop.)</li>
			  <li>Follow signs to the State Farm Arena exit after getting off the train.</li>
			  <li>Go up the escalator and veer left at street to walk around State Farm      Arena, turning right in front of the&nbsp;Georgia World Congress Center.</li>
			  <li>The hotel is on the next corner (see <a href="http://maps.google.com/maps?f=d&source=s_d&saddr=Philips+Dr&daddr=Andrew+Young+International+Blvd+NW&hl=en&geocode=FV0UAwIdyTL4-g%3BFa8dAwId3Dz4-g&gl=us&mra=dme&mrsp=1&sz=17&dirflg=w&sll=33.758474,-84.395471&sspn=0.003818,0.006899&ie=UTF8&ll=33.758474,-84.395471&spn=0.007635,0.013797&z=16" target="_blank">walking map</a>).</li>
		   </ol>
			<p>The <a href="http://www.anrdoezrs.net/links/7715686/type/dlg/http:/www.hotels.com/ho113333/omni-atlanta-hotel-at-cnn-center-atlanta-united-states/" target="_blank">Omni Hotel</a>&nbsp;at 190 Marietta Street SW is a short walk from <a href="https://martaguide.com/2013/01/09/domegwccphilips-arenacnn-center-marta-station/" title="Dome/GWCC/Philips Arena/CNN Center MARTA Station" target="_blank">GWCC/CNN MARTA station</a>, is connected to the CNN Center and is the closest hotel to the Mercedes-Benz Stadium. Amenities include an outdoor heated pool, fitness center and the restaurant Prime Meridian.</p>
			<p>Walking time is about six minutes. There&rsquo;s also a hotel entrance inside the CNN Center as an alternate route.
			</p>

	    </div>
  	</div>
	</div>

	  <a id="register"><div class="container">
        <div class="row">
            <div class="col-xs-12 section-title"><a href="http://www.cvent.com/d/d6qjtf/4W" target="blank" style="color: #fff">Register</a></div>
        </div>
    	</div>
	</a>

    <div class="container section-content-end">
      <div class="row">
      </div>
      <div class="row">
  		 <div class="col-md-12">
		   <h2 style="text-align: center">Registration Ends May 10, 2019</h2>
			 <p align="center" class="reg-button-large"><a href="http://www.cvent.com/d/d6qjtf/4W" target="_blank"><button class="btn btn-danger btn-lg reg-gray">Register Now</button></a></p>
               <br>
           <p style="text-align: center"><strong>Event Support:</strong><br>
Jenn Cannon: 470-481-8846 or <a href="mailto:MicroCorpMeetingHQ@ElevateIME.com">MicroCorpMeetingHQ@ElevateIME.com</a>	    </p>
	    </div>
    	</div>

	</div>


      <!-- Bootstrap core JavaScript
      ================================================== -->
      <!-- Placed at the end of the document so the pages load faster -->
      <script src="js/jquery.min.js"></script>
      <script>window.jQuery || document.write('<script src="css/vendor/jquery.min.js"><\/script>')</script>
      <script src="js/bootstrap.min.js"></script>
      <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
      <script src="js/ie10-viewport-bug-workaround.js"></script>
    </body>
  </html>
