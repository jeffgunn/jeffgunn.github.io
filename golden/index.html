<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0" />
    <title>Videos</title>

    <!-- Import Golden's Styles -->
    <link rel="stylesheet" href="http://www.goldenpeanut.com/css/gp.min.css?v=1.0.0.14">
    <!-- End Import Golden's Styles -->

    <!-- Styles for Golden -->
    <style media="screen">
    .vid-info {
      width: 90%;
      margin: 0 auto 40px auto;
      padding: 15px 0 0 0;
      text-align: center;
      color: #fff;
      font-family: AvenirLTStd-Light,"Helvetica Neue",Helvetica,Arial,sans-serif;
      font-size: 1em;
      line-height: 25px;
      background-color: #333d47;
      border-bottom: 5px solid #fec842;
      border-radius: 5px 5px 0 0;
    }
    .vid-info h2 {
      margin: 0 0 20px 0;
      color: #fec842;
      font-family: BrandonGrotesque-Bold,"Helvetica Neue",Helvetica,Arial,sans-serif;
      text-transform: titlecase;
    }
    .vid-social {
      margin: 25px 0 10px 0;
      font-size: .9em;
      color: #D6D8DA;
    }
    .vid-icon-social {
      display: block;
      margin-bottom: 30px;
      text-align: center;
    }
    .vid-icon-social img {
      display: inline-block;
      margin: 0 10px;
      width: 40px;
      opacity: 0.8;
    }
    .vid-icon-social img:hover {
      opacity: 1;
      cursor: pointer;
    }

    .vid-info .vid-embed {
      border-top: 1px solid #fec842;
      border-bottom: 1px solid #fec842;
    }
    .vid-container {
      width: 90%;
      margin: 40px auto;
    }
    .vid-title {
      margin: 0;
      padding-bottom: 10px;
      text-align: center;
      font-family: BrandonGrotesque-Bold,"Helvetica Neue",Helvetica,Arial,sans-serif;
      color: #393939;
      text-transform: uppercase;
      font-size: 1.6em;
      letter-spacing: .1em;
    }
    .vid-embed {
      position: relative;
      padding-bottom: 56.25%;
      max-width: 100%;
      height: 0;
      overflow: hidden;
    }
    .vid-embed iframe, .vid-embed object, .vid-embed embed {
      position: absolute;
      width: 100%;
      height: 100%;
      top: 0;
      left: 0;
    }
    .vid-content {
      margin: 0;
      padding: 40px 30px 25px 30px;
      text-align: left;
      font-family: AvenirLTStd-Light,"Helvetica Neue",Helvetica,Arial,sans-serif;
      font-size: 1em;
      line-height: 25px;
      background-color: #e5e9ec;
      background-image: url(vid-shadow-light.jpg);
      background-repeat: no-repeat;
      background-position: center top;
      background-size: contain;
      border-bottom: 5px solid #fec842;
    }
    .vid-content-feature {
      margin: 0;
      padding: 25px 30px 25px 30px;
      text-align: left;
      font-family: AvenirLTStd-Light,"Helvetica Neue",Helvetica,Arial,sans-serif;
      font-size: 1.1em;
      line-height: 25px;
      background-color: #4d565e;
      background-image: url(vid-shadow-dark.jpg);
      background-repeat: no-repeat;
      background-position: center top;
      background-size: contain;
    }
    .vid-seper {
      width: 95%;
      margin: 0 auto;
      border: 0;
      border-bottom: 1px solid #F1F2F4;
    }
    </style>
    <!--End Styles for Golden-->

  </head>
  <body style="padding: 20px 0;">
  <!--BEGIN CUSTOM HTML-->
  <style media="screen">
  .vid-info {
    width: 90%;
    margin: 0 auto 40px auto;
    padding: 15px 0 0 0;
    text-align: center;
    color: #fff;
    font-family: AvenirLTStd-Light,"Helvetica Neue",Helvetica,Arial,sans-serif;
    font-size: 1em;
    line-height: 25px;
    background-color: #333d47;
    border-bottom: 5px solid #fec842;
    border-radius: 5px 5px 0 0;
  }
  .vid-info h2 {
    margin: 0 0 20px 0;
    color: #fec842;
    font-family: BrandonGrotesque-Bold,"Helvetica Neue",Helvetica,Arial,sans-serif;
    text-transform: titlecase;
  }
  .vid-social {
    margin: 25px 0 10px 0;
    font-size: .9em;
    color: #D6D8DA;
  }
  .vid-icon-social {
    display: block;
    margin-bottom: 30px;
    text-align: center;
  }
  .vid-icon-social img {
    display: inline-block;
    margin: 0 10px;
    width: 40px;
    opacity: 0.8;
  }
  .vid-icon-social img:hover {
    opacity: 1;
    cursor: pointer;
  }

  .vid-info .vid-embed {
    border-top: 1px solid #fec842;
    border-bottom: 1px solid #fec842;
  }
  .vid-container {
    width: 90%;
    margin: 40px auto;
  }
  .vid-title {
    margin: 0;
    padding-bottom: 10px;
    text-align: center;
    font-family: BrandonGrotesque-Bold,"Helvetica Neue",Helvetica,Arial,sans-serif;
    color: #393939;
    text-transform: uppercase;
    font-size: 1.6em;
    letter-spacing: .1em;
  }
  .vid-embed {
    position: relative;
    padding-bottom: 56.25%;
    max-width: 100%;
    height: 0;
    overflow: hidden;
  }
  .vid-embed iframe, .vid-embed object, .vid-embed embed {
    position: absolute;
    width: 100%;
    height: 100%;
    top: 0;
    left: 0;
  }
  .vid-content {
    margin: 0;
    padding: 40px 30px 25px 30px;
    text-align: left;
    font-family: AvenirLTStd-Light,"Helvetica Neue",Helvetica,Arial,sans-serif;
    font-size: 1em;
    line-height: 25px;
    background-color: #e5e9ec;
    background-image: url(vid-shadow-light.jpg);
    background-repeat: no-repeat;
    background-position: center top;
    background-size: contain;
    border-bottom: 5px solid #fec842;
  }
  .vid-content-feature {
    margin: 0;
    padding: 25px 30px 25px 30px;
    text-align: left;
    font-family: AvenirLTStd-Light,"Helvetica Neue",Helvetica,Arial,sans-serif;
    font-size: 1.1em;
    line-height: 25px;
    background-color: #4d565e;
    background-image: url(vid-shadow-dark.jpg);
    background-repeat: no-repeat;
    background-position: center top;
    background-size: contain;
  }
  .vid-seper {
    width: 95%;
    margin: 0 auto;
    border: 0;
    border-bottom: 1px solid #F1F2F4;
  }
  </style>
  <div>
    <!-- Header -->
    <section class="vid-info">
      <h2>#RealPeopleRealNuts</h2>
      <div class="vid-embed"><iframe width="853" height="480" src="https://www.youtube.com/embed/5tcAv2G_o10?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe></div>
      <div class="vid-content-feature">Bringing stories and insight from the U.S. and the world’s most productive peanut and tree nut farming and manufacturing communities, Golden Peanut and Tree Nuts is committed to its people. Coma survival, the everyday struggle of a farmer’s wife, and the process of adopting four siblings are just a few of the journeys these men and women have undergone. Follow their stories with us.</div>
      <p class="vid-social">Follow their stories on Instagram and Facebook</p>
      <div class="vid-icon-social"><a href="https://www.instagram.com/goldenpeanuttreenuts/" target="blank" alt="Instagram"><img src="vid-instagram.svg" alt="Instagram"></a><a href="https://www.facebook.com/GoldenPeanutCompany/" target="blank" alt="Facebook"><img src="vid-facebook.svg" alt="Facebook"></a><div>
    </section>
    <!-- End Header -->

    <!-- Begin Video Template -->
    <!--
    <hr class="vid-seper">
    <section class="vid-container">
      <div class="vid-title">TITLE HERE</div>
      <div class="vid-embed"><iframe width="853" height="480" src="https://www.youtube.com/embed/LINKHERE?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe></div>
      <div class="vid-content">DESCRIPTION HERE</div>
    </section>
    -->
    <!-- End Video Template -->

    <hr class="vid-seper">
    <section class="vid-container">
      <div class="vid-title">Steward of the Land</div>
      <div class="vid-embed"><iframe width="853" height="480" src="https://www.youtube.com/embed/-31TJJq2GI4?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe></div>
      <div class="vid-content">For farmer Marty McLendon, producing a profitable crop and taking care of the earth that his peanuts and cotton grow in go hand in hand. He worked with agricultural engineers to develop a sustainable irrigation system to water his land that is now used and manufactured all over the world.</div>
    </section>
    <hr class="vid-seper">
    <section class="vid-container">
      <div class="vid-title">My Perfect Storm</div>
      <div class="vid-embed"><iframe width="853" height="480" src="https://www.youtube.com/embed/DwAxLDqCrMg?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe></div>
      <div class="vid-content">After surviving a coma, Golden Peanut and Tree Nuts employee Matt Miller had to start over from the ground up, learning how to cope with a brain injury and navigate school at an age when he expected to be well on his way in a career as a scientist.</div>
    </section>
    <hr class="vid-seper">
    <section class="vid-container">
      <div class="vid-title">Money Grows On Trees</div>
      <div class="vid-embed"><iframe width="853" height="480" src="https://www.youtube.com/embed/vsgNZv_nCYw?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe></div>
      <div class="vid-content">South African farmer Frikkie Nel’s plan to pay for his health care bills and retirement? Growing pecans.</div>
    </section>
    <hr class="vid-seper">
    <section class="vid-container">
      <div class="vid-title">Not for Sissies</div>
      <div class="vid-embed"><iframe width="853" height="480" src="https://www.youtube.com/embed/C9bYJpguQ5s?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe></div>
      <div class="vid-content">Lori Jordan has worked with Golden Peanut and Tree Nuts for 33 years – and she’s learned a thing or two. As a woman in a leadership position in the agricultural industry, her experience has been unique.</div>
    </section>
    <hr class="vid-seper">
    <section class="vid-container">
      <div class="vid-title">Like Father Like Son</div>
      <div class="vid-embed"><iframe width="853" height="480" src="https://www.youtube.com/embed/gx2MyK8TkOM?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe></div>
      <div class="vid-content">Clint White’s great-great grandfather settled in Northern Texas in 1890. Several generations later, Clint and his son Colby are still working the land. But they’ve put their own spin on it: hog training.</div>
    </section>
    <hr class="vid-seper">
    <section class="vid-container">
      <div class="vid-title">The Crazy Eight</div>
      <div class="vid-embed"><iframe width="853" height="480" src="https://www.youtube.com/embed/IJkx7A29X2Y?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe></div>
      <div class="vid-content">Golden Peanut and Tree Nuts employee Brandon Harrell talks about the everyday triumphs and travails of raising four adopted brothers and two teenage daughters.</div>
    </section>
  </div>

  <!--END CUSTOM HTML-->
  </body>
</html>
